### DISCUSSION TITLE
Attach a pdf to the calendar event

### DISCUSSION THREAD

**Discussion by mrc-bsllt** [add2cal/add-to-calendar-button#277] [Q&A] [answered]
As anticipated by the title, is it possible to attach a pdf to the event? Thanks!

**Comment by jekuer** [comment, marked as answer]
Hi @mrc-bsllt , 
unfortunately this is not supported. 
It would also not be possible since the calendars do not provide this functionality*. 
Best thing you can do is add a link to the PDF, which is hosted somewhere else.

--- 

\* when you see the option at, for example, the Google calendar, this is always connected to your account (and in the Google case, your GDrive), which is hosting the file then. In a generic setup (like with an event sharing button) this is not possible by design unless you, again, would host the file yourself and add a link to it. And that's what I suggested above :) .
